docs/changelog.md

```diff
@@ -32,7 +32,20 @@
 
 # Changelog
 
-## 21 October 2021 - Changes staged on develop
+## 16 February 2022
```

Comment:
We'll need to update this date again and think if we want to release the marking definitions milestone as part of v1.2